lib/beaker/dsl/install_utils.rb

```diff
@@ -478,11 +478,18 @@ def do_install hosts, opts = {}
             host['dist'] = "puppet-enterprise-#{version}-#{host['platform']}"
           elsif host['platform'] =~ /windows/
             version = host[:pe_ver] || opts['pe_ver_win']
+            should_install_64bit = !(version_is_less(version, '3.4')) and host.is_x86_64? and not host['install_32'] and not opts['install_32']
```

Comment:
Prefer `&&` to `and`

http://devblog.avdi.org/2010/08/02/using-and-and-or-in-ruby/